modules/login/src/main/resources/META-INF/resources/login/js/registration.js

```diff
@@ -0,0 +1,401 @@
+var registration = function($) {
+    var orgLookupData = {};
+    var bInitialized = false;
```

Comment:
Stylistically, I'm not a fan of prefixing variables with their type. It's an area we haven't really discussed but it looks like this the only variable with a prefix.